docs/pages/jpscore/jpscore_introduction.md

```diff
@@ -48,7 +48,13 @@ Taking the 7th demo as input, we run a simulation as follows:
  ./bin/jpscore  demos/corner_ini.xml
 ```
 
-which produces a trajectory file in the folder `results` in the same directory. This can be visualized with `jpsvis`
+It is possible to redirect the output and write it to a file like:
+
+```bash
+ ./bin/jpscore  demos/corner_ini.xml &> log_example.txt
+```
+
+This program call produces a trajectory file in the folder `results` in the same directory. This can be visualized with `jpsvis`
```

Comment:
I would keep the old text and add the logging information at the end.
My suggestion:
Information on processing steps, warnings and errors of `jpscore` are written to the console. This output can be redirected to a file by:

```bash
 ./bin/jpscore  demos/corner_ini.xml > log_example.txt
```

Reply:
I found out that "&>" is not working on Windows but ">" does. Could you please change this in the jpsreport inifile documentation as well?

Reply:
good suggestion. Adopted